frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwarePackageCard/SoftwarePackageCard.tsx

```diff
@@ -305,68 +349,124 @@ const SoftwarePackageCard = ({
     );
   };
 
+  const renderSelfServiceInfo = () => {
+    return (
+      <div className={`${baseClass}__badge`}>
+        <Icon
+          name="install-self-service"
+          size="small"
+          color="ui-fleet-black-75"
+        />
+        Self-service
+      </div>
+    );
+  };
+
+  const renderLabelInfo = () => {
+    const labels = softwarePackage?.labels_include_any?.length
+      ? softwarePackage.labels_include_any.map((label) => label.name)
+      : softwarePackage?.labels_exclude_any.map((label) => label.name) || [];
+
```

Comment:
TODO on subsequent PR: Add newly designed tooltip header https://fleetdm.slack.com/archives/C02A8BRABB5/p1723160752715049?thread_ts=1723059265.568719&cid=C02A8BRABB5

Reply:
Just added since I was back in the code

``` 
const tooltipLines = softwarePackage?.labels_include_any?.length
      ? ["Includes any:", ...labels]
      : ["Excludes any:", ...labels];
```